packages/palette/src/elements/Swiper/activeIndex.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Remaps the progress into the active index, given an array of `length`.
+ * @param progress number from 1 to 100
+ * @param length length of an array
+ */
+export const activeIndex = ({
+  progress,
+  length,
+}: {
+  progress: number
+  length: number
+}) => {
+  return Math.floor((progress * (length - 1)) / 100)
+}
```

Comment:
Should this be it's own file? I don't feel strongly about it either way, but I'd like to hear your perspective.

Reply:
Honestly, I just wanted a `utils.ts` or `helpers.ts` file in both Carousel and Swiper. 😄